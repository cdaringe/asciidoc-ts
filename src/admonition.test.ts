import { test } from "vitest";
import { toAST } from "./mod.js";

test("Admonition - basic", async ({ expect }) => {
  const input = `NOTE: This is an admonition.`;
  const result = toAST(input);
  expect(result).toMatchInlineSnapshot(`
    {
      "blocks": [
        {
          "admonitionType": "NOTE",
          "content": "This is an admonition.",
          "type": "Admonition",
        },
      ],
      "type": "Document",
    }
  `);
});
