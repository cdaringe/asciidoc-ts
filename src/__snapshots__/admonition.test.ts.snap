// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Admonition - basic 1`] = `
{
  "blocks": [
    {
      "admonitionType": "NOTE",
      "content": "This is an admonition.",
      "type": "Admonition",
    },
  ],
  "type": "Document",
}
`;
